### PR TITLE
document --log-config-file

### LINKED ISSUE #10657: release-21.1: cli: better UX, new flag `--log-config-file`
PR: https://github.com/cockroachdb/cockroach/pull/64948

From release notes:
> The new parameter `--log-config-file` simplifies the process of loading the logging configuration from a YAML file. Instead of passing the content of the file via the `--log` flag (e.g., `--log=$(cat file.yaml)`), it is now possible to pass the path to the file using `--log-config-file=file.yaml`. <br>**Note:** Each occurrence of `--log` and `--log-config-file` on the command line overrides the configuration set from previous occurrences. [#64948][#64948] {% comment %}doc{% endcomment %}

### PR DESCRIPTION
**Pull request by taroface**
Resolves #10657.

### CODE REVIEW

**Review by knz — APPROVED** [review]

File: v21.1/configure-logs.md [hunk 1]
```diff
@@ -19,24 +19,34 @@ The logging flags previously used with `cockroach` commands are now deprecated.
...
 
 ## YAML payload
 
-All log settings for a `cockroach` command are specified with a YAML payload, which you can pass with the `--log` flag in two ways:
+All log settings for a `cockroach` command are specified with a YAML payload in one of the following formats:
+
+- Block format, where each parameter is written on a separate line. For example, after creating a file `logs.yaml`, pass the YAML values with either `--log-config-file` or `--log`:
 
-- Block format, where each parameter is written on a separate line. For example, after creating a file `logs.yaml`:
+  {% include copy-clipboard.html %}
```
Comment (ianjevans) [inline v21.1/configure-logs.md:37]:
```suggestion
  {% include_cached copy-clipboard.html %}
```

A nit, but I always forget this.

File: v21.1/configure-logs.md [hunk 2]
```diff
@@ -19,24 +19,34 @@ The logging flags previously used with `cockroach` commands are now deprecated.
...
+  ~~~ shell
+  $ cockroach start-single-node --certs-dir=certs --log-config-file=logs.yaml
+  ~~~
 
+  {% include copy-clipboard.html %}
```
Comment (ianjevans) [inline v21.1/configure-logs.md:42]:
```suggestion
  {% include_cached copy-clipboard.html %}
```

File: v21.1/configure-logs.md [hunk 3]
```diff
@@ -19,24 +19,34 @@ The logging flags previously used with `cockroach` commands are now deprecated.
...
 	~~~ shell
 	$ cockroach start-single-node --certs-dir=certs --log="$(cat logs.yaml)"
 	~~~
 
 - Inline format, where all parameters are specified on one line. For example, to generate an `ops` log file that collects the `OPS` and `HEALTH` channels (overriding the file groups defined for those channels in the [default configuration](#default-logging-configuration)):
 
+  {% include copy-clipboard.html %}
```
Comment (ianjevans) [inline v21.1/configure-logs.md:49]:
```suggestion
  {% include_cached copy-clipboard.html %}
```

**Review by ianjevans — APPROVED** [review]
LGTM, just suggested using the cached copy tag.

**Review by taroface — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10690)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ianjevans)

___
*[v21.1/configure-logs.md, line 37 at r1](https://reviewable.io/reviews/cockroachdb/docs/10690#-MbDC7wkl_vvRGqK8rqF-r1-37:-MbHXkKZE2svcofHxBHK:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c0bc982b9c23f3cf330985370c7a115333bdffd6/v21.1/configure-logs.md#L37)):*
<details><summary><i>Previously, ianjevans (Ian Evans) wrote…</i></summary><blockquote>

```suggestion
  {% include_cached copy-clipboard.html %}
```

A nit, but I always forget this.
</blockquote></details>

Done.
___
*[v21.1/configure-logs.md, line 42 at r1](https://reviewable.io/reviews/cockroachdb/docs/10690#-MbDC7wkl_vvRGqK8rqF-r1-42:-MbHXkYo1TKujV25vGV_:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c0bc982b9c23f3cf330985370c7a115333bdffd6/v21.1/configure-logs.md#L42)):*
<details><summary><i>Previously, ianjevans (Ian Evans) wrote…</i></summary><blockquote>

```suggestion
  {% include_cached copy-clipboard.html %}
```
</blockquote></details>

Done.
___
*[v21.1/configure-logs.md, line 49 at r1](https://reviewable.io/reviews/cockroachdb/docs/10690#-MbDC7wkl_vvRGqK8rqF-r1-49:-MbHXkwC78VBwYKB8k-v:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c0bc982b9c23f3cf330985370c7a115333bdffd6/v21.1/configure-logs.md#L49)):*
<details><summary><i>Previously, ianjevans (Ian Evans) wrote…</i></summary><blockquote>

```suggestion
  {% include_cached copy-clipboard.html %}
```
</blockquote></details>

Done.

**Pull request merged**